ui/imports/shared/popups/SettingsDirtyToastMessage.qml

```diff
@@ -113,47 +116,68 @@ Rectangle {
         hoverEnabled: true
     }
 
-    RowLayout {
+    ColumnLayout {
         id: toastContent
-
-        anchors {
-            fill: parent
-            margins: 16
+        anchors.fill: parent
+        anchors.margins: Style.current.padding
+        spacing: Style.current.padding
+
+        RowLayout {
+            Layout.alignment: Qt.AlignHCenter
+
+            StatusBaseText {
+                id: changesDetectedTextItem
+                Layout.fillWidth: true
+                padding: 8
+                horizontalAlignment: Text.AlignHCenter
+                color: Theme.palette.directColor1
+                text: root.defaultChangesDetectedText
+            }
+
+            StatusButton {
+                id: cancelChangesButton
+                text: root.defaultCancelChangesText
+                enabled: !root.loading && root.active
```

Comment:
I'd assume the user is free to cancel anytime, irrespective of the `loading` flag

Reply:
in this case, if rpc completes successfully, the user will see the changes after pressing cancel

Reply:
And if it fails? The Cancel button should still revert right?

Reply:
We decided that the `Cancel` button will be disabled during the "save" action is in progress to avoid confusing situations for the user. It's agreed with `Ben` as well!

Reply:
OK